docker/base-deps/README.md

```diff
@@ -0,0 +1,23 @@
+## About
```

Comment:
Should users use this at all? If not let's have a pointer to the images they should be using.

Reply:
Nope, I'll leave pointers here & on `ray-deps`